server/routes/api/auth.js

```diff
@@ -0,0 +1,128 @@
+var router = require('express').Router(),
+    userModel = require('../../db/model/User.js'),
+    authMiddleware = require('../../middleware/auth.js');
+
+// Disable all non-post methods for /v1/auth
+router.use(function(req, res, next) {
+    if (req.method !== 'POST') {
+        res.status(405).send({
+            status: false,
+            message: "Method not allowed"
+        });
+    }
+    return next();
+});
+
+// Handles /v1/auth/login
+router.post('/login', function(req, res) {
+    // Check if the login request contains the email and password
+    if (req.body.email && req.body.password) {
+        // Lookup users with the email provided in the post body
+        userModel.find().byEmail(req.body.email).exec(function(err, model) {
```

Comment:
definitely the iOS dev in me again, but we could use `User` here instead of `userModel` to show a class method, not sure what's preferred in the node world here honestly